src/Tizen.NUI/src/public/Layouting/LayoutGroup.cs

```diff
@@ -65,30 +72,32 @@ public virtual void Add(LayoutItem childLayout)
         /// <summary>
         /// Remove all layout children.<br />
         /// </summary>
+        /// <since_tizen> 6 </since_tizen>
         public void RemoveAll()
         {
-            foreach( LayoutItem childLayout in _children )
+            foreach( LayoutItem childLayout in LayoutChildren )
             {
                 childLayout.ConditionForAnimation = ConditionForAnimation | TransitionCondition.Remove;
                 childLayout.Owner = null;
             }
-            _children.Clear();
+            LayoutChildren.Clear();
             // todo ensure child LayoutItems are still not parented to this group.
             RequestLayout();
         }
 
         /// <summary>
         /// From ILayoutParent
         /// </summary>
+        /// <since_tizen> 6 </since_tizen>
         public virtual void Remove(LayoutItem layoutItem)
```

Comment:
Parameters of method must have valid documentation. Method has 1 parameter, 0 are documented.